### PR TITLE
GraphQL - add account query

### PR DESCRIPTION
**Pull request by toyhammered**
# Checklist

- [ ] All files pass Rubocop
- [ ] Any complex logic is commented
- [ ] Any new systems have thorough documentation
- [ ] Any user-facing changes are behind a feature flag (or: explain why they can't be)

- [ ] All the tests pass
- [ ] Tests have been added to cover the new code

### CODE REVIEW

File: app/graphql/types/query_type.rb [hunk 1]
```diff
@@ -1,4 +1,12 @@
 class Types::QueryType < Types::BaseObject
+  field :my_account, Types::Account, null: true do
```
Comment (wopian) [inline app/graphql/types/query_type.rb]: `current_account`? `my_account` feels kinda odd when you're doing auth for other users. Same with description.

Comment (toyhammered) [reply app/graphql/types/query_type.rb]: `me` lololol

Comment (NuckChorris) [reply app/graphql/types/query_type.rb]: I agree with wopian, `current_account` is probably a better phrasing

Comment (toyhammered) [reply app/graphql/types/query_type.rb]: ok

**Review by NuckChorris — APPROVED** [review]

**Review by wopian — APPROVED** [review]

**Pull request merged**